pkg/api/apis/operators/v1alpha1/installplan_types.go

```diff
@@ -254,6 +254,37 @@ func NewStepResourcesFromCRD(crd *v1beta1.CustomResourceDefinition) ([]StepResou
 	return []StepResource{crdStep, aggregatedEditClusterRoleStep, aggregatedViewClusterRoleStep}, nil
 }
 
+// NewStepResourceForObject returns a new StepResource for the provided object
+func NewStepResourceFromObject(obj runtime.Object, name string) (StepResource, error) {
+	var resource StepResource
+
+	// set up object serializer
+	serScheme := runtime.NewScheme()
```

Comment:
maybe we should just build the scheme on `init` and use it here

Reply:
sounds good